src/sections/offering/price-info.tsx

```diff
@@ -0,0 +1,155 @@
+import { colors, fonts } from '@hedviginsurance/brand'
```

Comment:
Can we maybe rename this file (`price-info`) to `priceInfo`? i think we should be consequent with camel case or kebab case, and personally at least i prefer camel